streams-tests/shared/src/test/scala/zio/stream/experimental/ZStreamSpec.scala

```diff
@@ -2367,60 +2367,60 @@ object ZStreamSpec extends ZIOBaseSpec {
-        //             .take(4)
-        //             .runCollect
-        //         )(equalTo(Chunk("A", "A", "B", "B")))
-        //       ),
-        //       testM("short circuits after schedule")(
-        //         assertM(
-        //           ZStream("A", "B", "C")
-        //             .repeatElements(Schedule.once)
-        //             .take(3)
-        //             .runCollect
-        //         )(equalTo(Chunk("A", "A", "B")))
-        //       )
-        //     ),
-        //     testM("some") {
-        //       val s1 = ZStream.succeed(Some(1)) ++ ZStream.succeed(None)
-        //       s1.some.runCollect.either.map(assert(_)(isLeft(equalTo(None))))
-        //     },
-        //     testM("someOrElse") {
-        //       val s1 = ZStream.succeed(Some(1)) ++ ZStream.succeed(None)
-        //       s1.someOrElse(-1).runCollect.map(assert(_)(equalTo(Chunk(1, -1))))
-        //     },
-        //     testM("someOrFail") {
-        //       val s1 = ZStream.succeed(Some(1)) ++ ZStream.succeed(None)
-        //       s1.someOrFail(-1).runCollect.either.map(assert(_)(isLeft(equalTo(-1))))
-        //     },
+             suite("repeatElements")(
+               testM("repeatElementsWith")(
+                 assertM(
+                   ZStream("A", "B", "C")
+                     .repeatElementsWith(Schedule.recurs(0) *> Schedule.fromFunction((_) => 123))(
+                       identity,
+                       _.toString
+                     )
+                     .runCollect
+                 )(equalTo(Chunk("A", "123", "B", "123", "C", "123")))
+               ),
+               testM("repeatElementsEither")(
+                 assertM(
+                   ZStream("A", "B", "C")
+                     .repeatElementsEither(Schedule.recurs(0) *> Schedule.fromFunction((_) => 123))
+                     .runCollect
+                 )(equalTo(Chunk(Right("A"), Left(123), Right("B"), Left(123), Right("C"), Left(123))))
+               ),
+               testM("repeated && assertspaced")(
+                 assertM(
+                   ZStream("A", "B", "C")
+                     .repeatElements(Schedule.once)
+                     .runCollect
+                 )(equalTo(Chunk("A", "A", "B", "B", "C", "C")))
+               ),
+               testM("short circuits in schedule")(
+                 assertM(
+                   ZStream("A", "B", "C")
+                     .repeatElements(Schedule.once)
+                     .take(4)
+                     .runCollect
+                 )(equalTo(Chunk("A", "A", "B", "B")))
+               ),
+               testM("short circuits after schedule")(
+                 assertM(
+                   ZStream("A", "B", "C")
+                     .repeatElements(Schedule.once)
+                     .take(3)
+                     .runCollect
+                 )(equalTo(Chunk("A", "A", "B")))
+               )
+             ),
+             testM("some") {
```

Comment:
Something looks off in the suite structure here?

Reply:
I'm guessing this is because these individual operators only have a single test, so they aren't being grouped under a suite. I just uncommented them because they appear to have been implemented already. I checked and the structure is the same in the previous version as well.